### PR TITLE
Weather app

### PR DESCRIPTION
**Pull request by johannacatalinismith**
## Netlify link
https://todaysweather4u.netlify.app

### CODE REVIEW

**Comment by HIPPIEKICK** [comment]
The Netlify link doesn't seem to be working. Did you remember to specify that your code is in the /code folder when deploying?

**Comment by johannacatalinismith** [comment]
Hi! Sorry for the late replay, I've started my new job this week. Yes, that was it. It's now working :)

File: code/script.js [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+
+  //"new.date" is the timestamp from the 1970's. "*1000 is the turn the time into milliseconds, because js want's it like that."
+  const sunriseTime = new Date(weather.sys.sunrise * 1000);
+
+  sunrise.innerHTML = `Sunrise ${hourAndminutes(sunriseTime)}`;
+
+  const sunsetDate = new Date(weather.sys.sunset * 1000);
+  sunset.innerHTML = `Sunset ${hourAndminutes(sunsetDate)}`;
+
+  if (weather.weather[0].main == "Clouds") {
```
Comment (HIPPIEKICK) [inline code/script.js:44]: Consider putting `weather.weather[0].main` into a variable for better reusability

File: code/style.css [hunk 2]
```diff
@@ -0,0 +1,88 @@
+@import url("https://fonts.googleapis.com/css2?family=Montserrat:ital,wght@0,100..900;1,100..900&display=swap");
```
Comment (HIPPIEKICK) [inline code/style.css:1]: Great job implementing the design - looks super!

File: code/style.css [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+
+  .icon {
+    height: 100px;
+  }
+}
+
+@media (min-width: 1024px) {
+  body {
+
+    /* Here I tried to adjust margins with max-width instead of margin.
```
Comment (HIPPIEKICK) [inline code/style.css:83]: Looks great 👍

**Review by HIPPIEKICK — APPROVED** [review]
Good job with the weather app! Thanks for being quick with the link too 😄

**Pull request closed without merging**